docker-compose-db.yml

```diff
@@ -26,6 +26,24 @@ services:
     env_file:
       .env-db
 
+  nginx:
+    build:
+      context: ./nginx
```

Comment:
It would be better to change the context as well to db_nginx (folder related to it) to not mix up with the website for example